script.js

```diff
@@ -115,6 +115,10 @@ function renderEvents() {
   });
 }
 
+function test1() {
+  console.log("hello")
+}
```

Comment:
```suggestion
function test1() {
  console.log("Hello World!")
}
```
Hey I think that this should say Hello world instead.